specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -451,6 +451,62 @@
           }
         }
       }
+    },
+    "/custom/models/compose": {
+      "post": {
+        "tags": [
+          "Form"
+        ],
+        "summary": "Compose trained with labels models into one composed model.",
+        "description": "Compose request would include list of models ids.\r\nIt would validate what all models either trained with labels model or composed model.\r\nIt would validate limit of models put together.",
+        "operationId": "CustomModelsComposePost",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "composeRequest",
+            "in": "body",
+            "description": "Compose models",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ComposeRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "examples": {
+              "application/json": {
+                "error": {
+                  "code": "BadArgument",
+                  "innerError": {
+                    "requestId": "00000000-0000-0000-0000-000000000000"
+                  },
+                  "message": "Request body is invalid."
+                }
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Compose custom models": {
+            "$ref": "./examples/ComposeModels.json"
```

Comment:
do we have ComposeModels.json file in this PR?

Reply:
Yes